frontend/src/app/utils/index.tsx

```diff
@@ -30,7 +37,7 @@ export const showNotification = (toast, children) => {
 };
 
 export const showError = (
-  toast,
+  toast: Toast,
```

Comment:
Unrelated to this PR, but is there any reason we pass toast to this function and then to `showNotification` from all the components instead of just importing `toast` directly in this file?